fw/bno055_test/bno055_test.ino

```diff
@@ -21,19 +18,16 @@
    Connect SDA to analog 4
    Connect VDD to 3.3-5V DC
    Connect GROUND to common ground
-
-   History
-   =======
-   2015/MAR/03  - First release (KTOWN)
 */
 
 /* Set the delay between fresh samples */
-#define BNO055_SAMPLERATE_DELAY_MS (5)
+#define BNO055_SAMPLERATE_DELAY_MS (10)
 
 // Check I2C device address and correct line below (by default address is 0x29 or 0x28)
 //                                   id, address
 Adafruit_BNO055 bno = Adafruit_BNO055(55, 0x28);
-
+ESP8266Timer ITimer;
+volatile bool startRunning = false;
```

Comment:
Maybe change the variable name to something more meaningful, like "startSampling"